frontend/src/app/shared/lint.service.ts

```diff
@@ -49,4 +55,14 @@ export class LintService {
     }
     return result;
   }
+
+  lintYamlString(content: string): string | null {
```

Comment:
I think this whole method does not really belong here. The service should not interact with the UI via toastService, and the check whether it includes the workflow line should be in fulibWorkflows. Replacing tabs with spaces should also be done by the backend.

Reply:
Added it as a new issue for fulibWorkflows [issue8](https://github.com/fujaba/fulibWorkflows/issues/8) and removed it from the lint service.